data/india/maharashtra/ahmednagar/lsg/251539.json

```diff
@@ -0,0 +1 @@
+{"name": "Sangamner", "lsg_code": "251539", "district": "Ahmednagar", "state": "Maharashtra", "no_of_wards": 14, "wards": [{"ward_no": "10", "ward_name": "Sangamner(MC)- Ward No 10", "ward_id": "57513"}, {"ward_no": "12", "ward_name": "Sangamner(MC)- Ward No 12", "ward_id": "57515"}, {"ward_no": "6", "ward_name": "Sangamner(MC)- Ward No 6", "ward_id": "52541"}, {"ward_no": "5", "ward_name": "Sangamner(MC)- Ward No 5", "ward_id": "52540"}, {"ward_no": "4", "ward_name": "Sangamner(MC)- Ward No 4", "ward_id": "52539"}, {"ward_no": "3", "ward_name": "Sangamner(MC)- Ward No 3", "ward_id": "52538"}, {"ward_no": "2", "ward_name": "Sangamner(MC)- Ward No 2", "ward_id": "52537"}, {"ward_no": "13", "ward_name": "Sangamner(MC)- Ward No 13", "ward_id": "57516"}, {"ward_no": "14", "ward_name": "Sangamner(MC)- Ward No 14", "ward_id": "57517"}, {"ward_no": "11", "ward_name": "Sangamner(MC)- Ward No 11", "ward_id": "57514"}, {"ward_no": "7", "ward_name": "Sangamner(MC)- Ward No 7", "ward_id": "52542"}, {"ward_no": "8", "ward_name": "Sangamner(MC)- Ward No 8", "ward_id": "52543"}, {"ward_no": "1", "ward_name": "Sangamner(MC)- Ward No 1", "ward_id": "52536"}, {"ward_no": "9", "ward_name": "Sangamner(MC)- Ward No 9", "ward_id": "57512"}]}
```

Comment:
The ward name here seems to have ward no attached to it, this might ruin some of the data we are getting, also what is MC in each item ?

Reply:
Actually we have the data from http://lgdirectory.gov.in/ like that itself 😿
This is the CSV 👇📍 I think there should be some manual cleaning process in the CSVs 🧹
https://github.com/shrihari689/india-statewise-data/blob/master/csv_files/Maharashtra%20-%20Urban.csv
https://github.com/shrihari689/india-statewise-data/blob/master/csv_files/Maharashtra%20-%20PRI.csv

Reply:
Actually we have the data from http://lgdirectory.gov.in/ like that itself 😿
This is the CSV 👇📍 I think there should be some manual cleaning process in the CSVs 🧹
https://github.com/shrihari689/india-statewise-data/blob/master/csv_files/Maharashtra%20-%20Urban.csv
https://github.com/shrihari689/india-statewise-data/blob/master/csv_files/Maharashtra%20-%20PRI.csv